src/commands/selectSignImage.ts

```diff
@@ -0,0 +1,102 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCLI } from '../utils/checkCLI';
+import { execAsync } from '../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotationButton = 'Download Notation';
+const NotationURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notation not found: `;
+
+//transforms the key list string into an array of objects
+async function sortOutput(Key_String_Values: string ){
+    var dataArray = new Array();
+    var keyChoices = Key_String_Values.split("\n");
+    for (let i = 1; i < keyChoices.length; i++) { 
+      const values = keyChoices[i].trim().split(/ {2,} /); 
+      var itemObject; 
+        for (let j = 0; j < 1; j++) { 
```

Comment:
Is this loop only going to work when your list has two items?  Shouldn't this go through your whole list?

Reply:
With the way this loop is set up we are only taking the key names and we are capturing the whole list.